src/components/pageTitle.tsx

```diff
@@ -72,10 +72,10 @@ const TitleContainer = styled.div<{
     width: max-content;
   `)}
 
-  ${applyMediaQueryLg(css`
+  @media only screen and (min-width: 1220px) {
```

Comment:
i would prefer to use the same media query everywhere...

@Tuukkaa would it make sense to set the breakpoint for desktop to 1220px everywhere?

Reply:
@judithhartmann What is the benefit of using the same media query everywhere? Personally, I think that content should define where is the best place to "break" instead of having global breakpoints that are forced for all components. Every component could have custom breakpoints to adjust the content. 

Changing the desktop breakpoint to 1220px would break other layouts because some elements become too large. What if we add a fourth breakpoint, e.g., desktopLarge?